tools/src/main/java/com/powsybl/tools/autocompletion/commonscli/CommonsCliAdapter.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.tools.autocompletion.commonscli;
+
+import com.powsybl.tools.autocompletion.BashCommand;
+import com.powsybl.tools.autocompletion.BashOption;
+import org.apache.commons.cli.Options;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Adapter for apache commons CLI options
+ *
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at gmail.com>
+ */
+public final class CommonsCliAdapter {
+
+    private CommonsCliAdapter() {
+    }
+
+    public static List<BashOption> adaptOptions(Options commonsCliOptions) {
```

Comment:
As we do not intend to support multiples cli lib, maybe this class is not needed and we can directly move theses utilities Options to BashOption construction to BashOption it self?